test/hooks/index.js

```diff
@@ -29,6 +29,35 @@ describe('hooks', () => {
 			.then(bundle => {});
 	});
 
+	it('allows to read and modify output options in the outputOptions hook', () => {
+		return rollup
+			.rollup({
+				input: 'input',
+				treeshake: false,
+				plugins: [
+					loader({ input: `alert('hello')` }),
+					{
+						renderChunk(code, chunk, options) {
+							assert.strictEqual(options.banner, 'new banner');
+							assert.strictEqual(options.format, 'cjs');
```

Comment:
It would be nice to verify that not only the options are updated, but the code was actually rendered using the updated options by inspecting the generated code.

For the latter, I think it would be best to inspect the output of `bundle.generate`